### ISSUE TITLE
Respawn on highlander

### ISSUE THREAD

**Issue by Fr-Dae** [unitystation/unitystation#8909]
**Environment**

**- Game Version:** 22-06-17-04
**- OS Version:** Lubuntu 18.04.6x64
**- Linux version**  `5.17.0-14.1-liquorix-amd64 (steven@liquorix.net) (gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0, GNU ld (GNU Binutils for Ubuntu) 2.30) #1 ZEN SMP PREEMPT liquorix 5.17-18ubuntu1~bionic (2022-06-10)`
**- Server:** Locale

**Detailed description of bug**
**Context**
HOP , in Fallstation, 

wizard, 
used a book of magic.
ritual, "magic" makes Respawn as a hightlander.

**Comment by corp-0** [comment]
this issue is hard to understand. Can you rephrase what you meant or close if this isn't a problem anymore?

**Comment by Fr-Dae** [comment]
Ritual "magic" is no longer present.
only "summon gun" and another sound are present and working properly
